### DISCUSSION TITLE
support renaming/moving files

### DISCUSSION THREAD

**Discussion by DetachHead** [KotlinIsland/basedmypy#153] [Ideas]
maybe a command line option to rename a file which  updates all instances of it in the baseline

**Comment by KotlinIsland** [comment]
Would be better than re-baselineing because it wouldn't add new baseline entries, just move the old ones

What about renaming classes and functions?

**Comment by KotlinIsland** [comment]
The suggestion https://github.com/KotlinIsland/basedmypy/discussions/134#discussioncomment-2138396 of using version control information could be applied here.
